client/blocks/jitm/templates/sidebar-banner.jsx

```diff
@@ -36,6 +36,7 @@ export default function SidebarBannerTemplate( {
 			compact
 			event={ displayName }
 			forceHref={ true }
+			forceDisplay={ true }
```

Comment:
What do you think about maybe leaving a comment to clarify why we're passing `forceDisplay = true`? I could see it being a little bit confusing.